packages/kbn-shared-ux-components/src/index.ts

```diff
@@ -41,6 +41,11 @@ export const ExitFullScreenButton = withSuspense(LazyExitFullScreenButton);
  */
 export const ToolbarButton = withSuspense(LazyToolbarButton);
 
+/**
+ * An example of the solution toolbar button
+ */
+export { AddFromLibraryButton } from './toolbar/buttons/add_from_library/add_from_library';
+
```

Comment:
You should be able to import this from `./toolbar` directly, since you are already exporting it from there.

Reply:
Yussss thank you 🙌🏻  [63d8c31](https://github.com/elastic/kibana/pull/127605/commits/63d8c31f0de1ff81e9cff47f29178a03f01fc9d7) for changes